rasa/core/run.py

```diff
@@ -224,12 +224,11 @@ async def load_agent_on_start(
     endpoints: AvailableEndpoints,
     remote_storage: Optional[Text],
     app: Sanic,
-    loop: Text,
```

Comment:
I think we can't remove this as `Sanic` passes the loop when running this function. Pretty sure that this would lead to an error otherwise

Reply:
:sweat_smile: Adding it back...